### ISSUE TITLE
Add input gesture menu has no cancel option

### ISSUE THREAD

**Issue by XLTechie** [nvaccess/nvda#12285]
### Steps to reproduce:

1. Open Input Gestures dialog (NVDA+n, p, n).
2. Open a category, such as "Configuration".
3. Move to an item, such as "Reports the name of the current NVDA configuration profile".
4. Add any gesture, by pressing Alt+A. For example, use NVDA+Shift+Alt+C.

### Actual behavior:

The context menu appears, containing options for desktop layout, laptop layout, and all layouts.
However, there is no option to cancel the addition of this gesture.
The only way out is to apply the gesture, and then cancel the entire input gesture dialog, which aborts the save operation.
This action loses any other gestures the user may have already added/removed/altered.

I consider this a UX bug.

### Expected behavior:

There is a fourth option on the menu: `Cancel`.

#### NVDA

Installed or portable.
All versions as of 4/9/2021 (Currently on alpha-22295,49a85788)

#### Windows version:

Windows 10, also applies to Windows 7

**Comment by XLTechie** [comment]
I view this as a stepping stone to a solution for #11419, and the idea for adding a Cancel option is one of the ideas discussed there (although I didn't remember that until posting this and rereading that).

I'm fine if someone wants to merge these issues, but that one is particularly keyboard focused, where this starts with adding the cancel option which could potentially then resolve the keyboard question.

**Comment by CyrilleB79** [comment]
Yes, the two GitHub issues are two solutions for the same problem.

Note that, you also write:
> The context menu appears, containing options for desktop layout, laptop layout, and all layouts.
> However, there is no option to cancel the addition of this gesture.
> The only way out is to apply the gesture, and then cancel the entire input gesture dialog, which aborts the save operation.
> This action loses any other gestures the user may have already added/removed/altered.

Another solutions is to select the shortcut just added and remove it if you remember it. This avoid to lose all modifications made previously in the window. But it is not user-friendly either.

**Comment by XLTechie** [comment]
> Another solutions is to select the shortcut just added and remove it if you remember it. This avoid to lose all modifications made previously in the window.
> But it is not user-friendly either.

True, although you may have done something with the just assigned gesture, that 
compromises your ability to delete that gesture (like reassigning alt+r or 
similar).
